### PR TITLE
New endpoints added

### PR DESCRIPTION
**Pull request by vutukuriajay2241**
Updated in documentation, miner with api.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -78,9 +78,9 @@ Below are a list of ASCT+B tables supported by the Reporter,
 
 Click [here](https://ccf-asct-reporter.netlify.app/docs) to view the documentation.
 
-### Data Miner
+### ASCTB API
 
-The reporter is also accompanied by a backend server ASCT+B Data Miner. In the event of Google Sheets blocking requests by the Reporter or any error, the Reporter fetches the data from the Data Miner which runs a simple Node script to fetch the data from Google Sheets. By supplying the SheetId and GID to the Data Miner, the data from the flattened tables can be retrieved. If for some reason the Data Miner also fails to retrieve the data, the Reporter falls to its system cache, which contains a snapshot of the flattened tables. 
+The reporter is also accompanied by a backend server ASCT+B API. In the event of Google Sheets blocking requests by the Reporter or any error, the Reporter fetches the data from the ASCTB API which runs a simple Node script to fetch the data from Google Sheets. By supplying the SheetId and GID to the ASCTB API, the data from the flattened tables can be retrieved. If for some reason the ASCTB API also fails to retrieve the data, the Reporter falls to its system cache, which contains a snapshot of the flattened tables.
```
Comment (bherr2) [inline README.md:83]: In text, where possible, always keep the same string "ASCT+B API"

File: README.md [hunk 2]
```diff
@@ -92,7 +92,7 @@ This will either return the data, or will return a `500` HTTP code.
 
 #### Deploying 
 
-Since the ASCT+B Data Miner is a dynamic script, Heroku has been used to deploy the server. Heroky is free to use for this usage. Deploying on Heroku requires the `data-miner` folder to have an additional git remote. To deploy the latest changes to the Heroku cloud,
+Since the ASCT+B API is a dynamic script, Heroku has been used to deploy the server. Heroky is free to use for this usage. Deploying on Heroku requires the `data-miner` folder to have an additional git remote. To deploy the latest changes to the Heroku cloud,
```
Comment (bherr2) [inline README.md:95]: Rename data-miner dir to asctb-api

File: projects/v2/src/app/static/url.ts [hunk 3]
```diff
@@ -3,7 +3,7 @@ import { environment } from 'projects/v2/src/environments/environment';
 const PROD_URL = 'https://astb-miner-v2.herokuapp.com/v2';
```
Comment (bherr2) [inline projects/v2/src/app/static/url.ts:3]: You should be able to remove PROD_URL and DEV_URL constants

File: projects/v2/src/environments/environment.prod.ts [hunk 4]
```diff
@@ -1,5 +1,6 @@
 export const environment = {
   production: true,
   tag: '',
-  googleAnalyticsId: 'UA-136932895-5'
+  googleAnalyticsId: 'UA-136932895-5',
+  apasctbApi: 'https://asctb-api.herokuapp.com/v2'
```
Comment (bherr2) [inline projects/v2/src/environments/environment.prod.ts:5]: What does the ap stand for? Either way let's call it asctbApiUrl.
Also, remove the /v2 and add it elsewhere where appropriate. We will likely be adding new /vX and other urls to that API soonish (ie for exporting bits to graph formats, etc).

**Review by bherr2 — CHANGES_REQUESTED** [review]

File: projects/v2/src/environments/environment.prod.ts (hunk 4 above)
Comment (vutukuriajay2241) [reply projects/v2/src/environments/environment.prod.ts:5]: Done

File: projects/v2/src/app/static/url.ts (hunk 3 above)
Comment (vutukuriajay2241) [reply projects/v2/src/app/static/url.ts:3]: Done

**Review by bherr2 — APPROVED** [review]

**Pull request merged**